### PR TITLE
[MBL-17182][Student] Offline content fixes

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: 
- Each bug is listed in the ticket.
- The best way to review is commit by commit, each commit represents one bugfix.

refs: MBL-17182
affects: Student
release note: none

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/features/offline/sync/FileSync.kt [hunk 1]
```diff
@@ -20,7 +20,6 @@ package com.instructure.pandautils.features.offline.sync
 
 import android.content.Context
 import android.net.Uri
-import android.util.Log
```
Comment (hermannakos) [inline libs/pandautils/src/main/java/com/instructure/pandautils/features/offline/sync/FileSync.kt:23]: Nice

**Review by hermannakos — APPROVED** [review]
QA + 1

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

**Review by hermannakos — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

**Pull request merged**